packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -345,10 +349,16 @@ export default defineComponent({
       return this.resource.owner?.displayName
     },
     resourceSize() {
+      if (this.resource.id === this.currentFolder.id) {
```

Comment:
For places where we don't have a current folder, e.g. search results.

```suggestion
      if (this.resource.id === this.currentFolder?.id) {
```